_episodes/02-image-basics.md

```diff
@@ -91,13 +91,14 @@ First, the necessary imports:
 
 ~~~
 """
- * Python libraries for learning and performing image processing.*
+ * Python libraries for learning and performing image processing.
+ *
 """
 import numpy as np
-import skimage.io
-import skimage.viewer
```

Comment:
Removing because, unless I missed it, `skimage.viewer` never gets used/mentioned in the rest of the lesson.